cilium/cmd/config.go

```diff
@@ -111,3 +104,62 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 		Fatalf("Unable to change agent configuration: %s\n", err)
 	}
 }
+
+func printConfigurations(cfgStatus *models.DaemonConfigurationStatus) {
+	if command.OutputJSON() {
+		if listReadOnlyConfigurations {
+			if err := command.PrintOutput(cfgStatus.DaemonConfigurationMap); err != nil {
+				Fatalf("Cannot print json output: %v", err)
```

Comment:
"Cannot print json output" doesn't really mean much to the user. What really is happening is that we cannot show the configuration:

```suggestion
				Fatalf("Cannot show configuration: %v", err)
```